sources/osgShader/ShaderGenerator.js

```diff
@@ -215,10 +214,13 @@ ShaderGenerator.prototype = {
                 return cache;
             }
 
+            this._osgShader = this._osgShader || require( 'osgShader/osgShader' );
+            var doTimeCompilation = this._osgShader.enableShaderCompilationTiming;
```

Comment:
same thing than the debug/info log level. if ?log=info is not enough then yeah maybe we need a global flag.